lmfdb/genus2_curves/web_g2c.py

```diff
@@ -534,7 +537,20 @@ def local_table(D,N,tama,bad_lpolys):
     loctab.extend(['</tbody>', '</table>'])
     return '\n'.join(loctab)
 
+def ptscmp(a,b):
```

Comment:
This is not Python 3 compatible.  See "The cmp Argument" section of https://portingguide.readthedocs.io/en/latest/comparisons.html.

Reply:
Converted cmp to key

Reply:
You can see this in action on
https://red.lmfdb.xyz/Genus2Curve/Q/792079/a/792079/1
(or any curve with lots of rational points)